src/components/changes-tab/ChangesTabPage.vue

```diff
@@ -59,44 +59,28 @@ export default {
   watch: {
     model() {
       this.reloadDeviceChanges()
+      this.loadMoreChanges()
     },
     changes() {
```

Comment:
@luk1337 do you know what this is for?

Reply:
seems like it just does some additional work after `changes` is changed:

* filter/group for selected device
* trigger next page loading if scrolled to bottom

Reply:
that's such a convoluted way for sorting device specific changes ^.^ anyways my changes should be fine i think